handler/core/src/test/java/io/knotx/fragments/handler/consumer/EventLogConverterTest.java

```diff
@@ -43,7 +44,7 @@ void fillWithEmptyLog() {
 
     NodeExecutionData result = tested.getExecutionData(NODE_ID);
 
-    assertEquals(LoggedNodeStatus.UNPROCESSED, result.getStatus());
+    Assertions.assertEquals(LoggedNodeStatus.UNPROCESSED, result.getStatus());
```

Comment:
Is this an intentional change? I think it's a standard to import static these assertions

Reply:
Sure, we should use static imports.